eel/eel-stock-dialogs.c

```diff
@@ -213,7 +231,11 @@ timed_wait_callback (gpointer callback_data)
 		      "secondary-text", _("You can stop this operation by clicking cancel."),
 		      NULL);
 
-    gtk_dialog_add_button (GTK_DIALOG (dialog), button, GTK_RESPONSE_OK);
+    if (button == "process-stop")
```

Comment:
Shouldn't strings be compared via `g_strcmp0` or similar functions?
Or just use only check for `wait->cancel_callback` to set both button text and icon...